monitoring/nethealth.go

```diff
@@ -349,11 +316,27 @@ func fetchNethealthMetrics(ctx context.Context, addr string) ([]byte, error) {
 	//      # TYPE nethealth_echo_timeout_total counter
 	//      nethealth_echo_timeout_total{node_name="10.128.0.96",peer_name="10.128.0.70"} 37
 	//      nethealth_echo_timeout_total{node_name="10.128.0.96",peer_name="10.128.0.97"} 0
-	resp, err := client.Get(ctx, client.Endpoint("metrics"), url.Values{})
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, "http://unix/metrics", nil)
```

Comment:
nit: do we have all clients using go 1.13 where it was introduced?

Reply:
Depends if someone decides to backport. AFAIK 7.0 on gravity/planet are built with go 1.13, which are the only intended backport targets at present.